scalameta/contrib/shared/src/main/scala/scala/meta/contrib/implicits/ExtractExtensions.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.contrib.implicits
+
+import scala.meta.contrib._
+
+trait ExtractExtensions {
+
+  /**
+    * The motivation for this implicit enrichment is to extract values out of trees
```

Comment:
I guess we can. I more thought its a point of contact for people using "Jump to symbol" to try and find out what this method does